config/crds/apis.kcp.dev_apibindings.yaml

```diff
@@ -69,6 +69,12 @@ spec:
                   required:
                   - resource
                   type: object
+                  x-kubernetes-validations:
+                  - message: if claim for core group must only be one of configmaps,
```

Comment:
core group does not really matter here. You can claim Services or Pods. I think the only restriction we can and should have is about the identity.

Reply:
We may want to update this in the future. But if we are providing things like RBAC, that will not be the core group and will not have an identityHash. I believe that for now let's keep as is (most restrictive) and we can open it up as that path is usually more accessible. Thoughts?